inference.py

```diff
@@ -0,0 +1,106 @@
+# We would like to thank and acknowledge jfzhang95 for the DeepLabV3+ module 
+# as well as a template for metrics and the training pipeline. 
+# His code repository can be found here:
+# https://github.com/jfzhang95/pytorch-deeplab-xception
+
+from models.deeplab.train import *
+from models.deeplab.evaluate import *
+import argparse
+import os
+import time
+import rasterio
+
+def main():
```

Comment:
This code is great! However, I would appreciate if you could merge this in with `run_deeplab.py`. I understand I had a `train_deeplab.py` so it did not seem fitting to add code there. However, I have just made a rename and believe this code would be best suited in `run_deeplab.py`. 

If you could move your additional flags to after the `--boundary-kernel-size` flag, that would be awesome. And then add the `handle_inference` method.